dbms/src/test/java/org/polypheny/db/crossmodel/RelationalOnLpgTest.java

```diff
@@ -128,7 +126,7 @@ public void simpleProjectTest() {
 
 
     @Test
-    @Category(Neo4jExcluded.class) // see simpleProjectTest method
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
We scaped that method as one should use the json syntax and not this mix between SQL and Mongo. Removed.